bundles/af-core/src/test/resources/form/dropdown/exporter-dropdown-1.json

```diff
@@ -15,9 +15,18 @@
   "readOnly": false,
   "enforceEnum": true,
   "enumNames": [
-    "m",
-    "f",
-    "o"
+    {
```

Comment:
we should have some test cases where enumNames are array of primitives as we must support it.

Reply:
in core-components we are returning it as an array of objects,
how are you suggesting to do this.
This is backward compatible since there is no change in saving of data in crx